packages/internal/src/files.ts

```diff
@@ -103,8 +114,8 @@ export const findRouteHooksSrc = (cwd: string = getPaths().web.src) => {
   })
 }
 
-export const findPrerenderedHtml = (cwd = getPaths().web.dist) =>
-  fg.sync('**/*.html', { cwd, ignore: ['200.html', '404.html'] })
+// export const findPrerenderedHtml = (cwd = getPaths().web.dist) =>
+//   fg.sync('**/*.html', { cwd, ignore: ['200.html', '404.html'] })
```

Comment:
Why did you comment out this?

Reply:
It's not used anywhere in the project. Some other files do import this function, but the imports are commented out:
> // NOTE: This function was copied from @redwoodjs/internal/dist/files to avoid depending on @redwoodjs/internal.